gcsfs/cli/gcsfuse.py

```diff
@@ -0,0 +1,21 @@
+from gcsfs.gcsfuse import FUSE, GCSFS
+import click
+
+
+@click.command()
+@click.argument('bucket', type=str, required=True)
+@click.argument('mount_point', type=str, required=True)
+@click.option('--token', type=str, required=False, default=None,
+              help="Token to use for authentication")
+@click.option('--project', type=str, required=False, default='',
```

Comment:
Specifically, google calls this Project ID (as opposed to Project Name, which had no character restrictions).